brd-ios/breadwallet/EQNetworking/KYCAPI.swift

```diff
@@ -10,6 +10,17 @@ enum KYCEndpoints: String, URLType {
     case personalInformation = "pi?%@"
     case uploadSelfieImage = "upload?type=SELFIE%@"
     case uploadFrontBackImage = "upload?type=ID%@"
+    case login = "auth/login%@"
+    
+    var url: String {
+        return String(format: Self.baseURL, rawValue)
+    }
+}
+
+enum KYCLoginEndpoints: String, URLType {
+    static var baseURL: String = "https://one-dev.moneybutton.io/blocksatoshi/one/auth/%@"
```

Comment:
the url is now extracted to .env settings...
think if you pull MIT it should already fix it

Reply:
thanks for letting us know